pep/node/server.js

```diff
@@ -18,7 +18,7 @@ module.exports = function(emberDistPath) {
         fastbootMiddleware({
             distPath: emberDistPath,
             //resilient mode = true swallows rendering errors and returns a 200 w/the default index.html
-            resilient: false
+            resilient: true
```

Comment:
What are we trying to fix with this? Initially this looks like its just going to hide the actual errors from us when we would want to fix the root cause?

Reply:
yes and no. this will (theoretically) be a safety net for errors/failed api requests that we don't account for, and the user will get served the traditional non-fastboot rendered index page, and when the app renders client-side, it would produce the same errors, but within the app UI. This is mainly to protect against the hard crashed response state where the user just sees a blank white page with the `{ error: "Internal server error"}` json that apigateway spits back normally if Fastboot fails to render the page. We never want the user to get that in prod.

Also note that this change only applies when running the prod fastboot server, so during dev if there's fastboot related errors they wouldn't be hidden and we should still make sure we are handling them. I'm still looking into the specific error thats causing the white error pages for us now, but wanted to try adding this as well as an extra layer of protection so the app more gracefully degrades to a traditional non-server rendered SPA in cases were fastboot just bombs out completely (but yes i agree these errors should still be tracked and handled at their source when possible).